### PR TITLE
{bio}[GCCcore/11.3.0,foss/2022a] vcflib v1.0.9, WFA2 v2.3.3 w/ R 4.2.1

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4001.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/d80fcdf1afbc30d6a88b50f55ba4e712 for a full test report.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/v/vcflib/vcflib-1.0.9-foss-2022a-R-4.2.1.eb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+postinstallcmds = ["cp -r %(builddir)s/%(name)s-%(version)s/scripts %(installdir)s"]
+
+modextrapaths = {'PATH': ['scripts']}
+
+sanity_check_paths = {
+    'files': [
+        'bin/vcffilter',
+        'bin/vcfcombine',
+        'lib/libvcflib.a',
+        'lib/pyvcflib.cpython-310-x86_64-linux-gnu.so',
```
Comment (boegel) [inline easybuild/easyconfigs/v/vcflib/vcflib-1.0.9-foss-2022a-R-4.2.1.eb:71]: I wouldn't check for this file specifically due to the messy filename (which is CPU-type specific, won't work on Arm), and check whether `import pyvcflib` works instead (probably needs adding `lib` to `$PYTHONPATH` via `modextrapaths`?).

Also, no need to spread this across multiple lines then?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node4005.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/7f0bf5a91c7e1ed864ff565888acd857 for a full test report.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/aab5d84fc07fb304add742b0fb45dfa4 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**